src/makeInfinityAdder.js

```diff
@@ -1,10 +1,27 @@
 'use strict';
 
 /***
+ *
  * @return {function}
  */
+
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const adder = (arg = 0) => {
+    if (!arg) {
```

Comment:
The check for `!arg` will fail for a valid numeric argument `0`. Since the function should only return the sum when called with no arguments, consider explicitly checking for `undefined` instead.